ui/lib/apps/ClusterInfo/pages/List.module.less

```diff
@@ -0,0 +1,14 @@
+@import '~antd/es/style/themes/default.less';
+
+.sticky_tabs_header {
+  padding-left: @padding-page; // 48px
+  height: @padding-page; // 48px
+  margin-bottom: @padding-md; // 16px
+  border-bottom: 1px solid @gray-4;
```

Comment:
Although `1px` and `solid` also have variables, I think here using the value may be more direct and clear.